platform/ios/src/MGLMapView.mm

```diff
@@ -4670,15 +4670,25 @@ - (void)applyIconIdentifier:(NSString *)iconIdentifier toAnnotationsWithImageReu
 
 #pragma mark - User Location -
 
+- (void)setLocationManager:(id<MGLLocationManager>)locationManager
+{
+    _locationManager = locationManager;
+    _locationManager.delegate = self;
```

Comment:
The documentation comment for this property states that setting the property to `nil` causes the map view to use the CLLocationManager implementation. Does that happen immediately, or upon the next location update?

Reply:
When the location manager is set to nil is going to use the default option the next time the `showsUserLocation` is set to yes.

Reply:
So MGLMapView would continue to serve as the old location manager’s delegate until then, without stopping that location manager? That could result in MGLMapView and MGLMapViewDelegate responding to doubled-up or contradictory location updates.